pkg/client/powervs_client.go

```diff
@@ -292,37 +306,79 @@ func (p *powerVSClient) GetInstances() (*models.PVMInstances, error) {
 	return p.InstanceClient.GetAll()
 }
 
-func (p *powerVSClient) GetCloudServiceInstances() ([]bluemixmodels.ServiceInstanceV2, error) {
-	var instances []bluemixmodels.ServiceInstanceV2
-	svcs, err := p.ResourceClient.ListInstances(controllerv2.ServiceInstanceQuery{
-		Type: "service_instance",
-	})
-	if err != nil {
-		return svcs, fmt.Errorf("failed to list the service instances: %v", err)
-	}
-	for _, svc := range svcs {
-		if svc.Crn.ServiceName == PowerServiceType {
-			instances = append(instances, svc)
+func (p *powerVSClient) GetCloudServiceInstances() ([]resourcecontrollerv2.ResourceInstance, error) {
```

Comment:
is this used anywhere? because this will return all the service instances in the machine like cos, cis etc.. so wondering if we can restrict the list just for the power catalog type

@dharaneeshvrd we have done something similar in the hyper shift right?

Reply:
https://github.com/openshift/hypershift/blob/af2f0b13103f9ac1373b4abcd0b83a2870dac81c/cmd/infra/powervs/validate.go#L53
PowerVS serviceID: abd259f0-9990-11e8-acc8-b9f54a8f1661
PowerVS planID: f165dd34-3a40-423b-9d95-e90a23f724dd